repoaudit/repoaudit/__init__.py

```diff
@@ -34,6 +34,38 @@
     )
 )
 
+apt_sources_option = click.option(
+    "--apt-source",
+    "-a",
+    required=False,
+    help=(
+        "Supply apt sources.list file. When provided, all entries"
+        "will be parsed for repo urls and their respective dists."
+    )
```

Comment:
Add this: `type= click.File("r")` to this option. This will automatically handle the file for you.